packages/yarnpkg-nm/sources/hoist.ts

```diff
@@ -46,11 +46,14 @@
  * until you run out of candidates for current tree root.
  */
 type PackageName = string;
-export type HoisterTree = {name: PackageName, identName: PackageName, reference: string, dependencies: Set<HoisterTree>, peerNames: Set<PackageName>, hoistPriority?: number, isWorkspace?: boolean};
+export enum HoisterDependencyKind {
+  REGULAR, WORKSPACE, PORTAL,
```

Comment:
Is there any value to calling it `HoisterDependencyKind.PORTAL` instead of `HoisterDependencyKind.EXTERNAL_SOFT_LINK` since the check is still for external soft links?

Reply:
Renamed